test/unittests/statetest_loader_test.cpp

```diff
@@ -117,7 +117,9 @@ TEST(statetest_loader, load_minimal_test)
                 "value": null,
                 "nonce" : "0"
             },
-            "post": {},
+            "post": {
+                "Cancun": []
```

Comment:
Do EEST have multiple entries here?

Reply:
Hm, no, it seems `"Cancun"` and `"Prague"` keys would get split into two tests. Do you think this warrants removing the [tests: Support varying BlockInfo in statetests](https://github.com/ethereum/evmone/pull/1118/commits/1ddc15395b0776132eaf0269f8ce9a01b78651e0)? And just using the first entry and asserting there is only one?

Reply:
No, I think the change is fine. We rather should push for the test format update long term.

Reply:
There is also one change I have in mind: we should split `BlockInfo` into multiple structs: data from tests, computed values, data needed for EVM execution.

Reply:
> No, I think the change is fine. We rather should push for the test format update long term.

Meanwhile, I've tried it. It is only marginally smaller, not worth it

> There is also one change I have in mind: we should split BlockInfo into multiple structs: data from tests, computed values, data needed for EVM execution.

I presume not in this PR?

> Do you want to split out the BlockInfo change?

I was not planning to squash it, so it remains as a separate commit, ok?